src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    const defaultCompare = (el1, el2) => {
+      const itemOne = `${el1}`;
+      const itemTwo = `${el2}`;
+
+      return itemOne > itemTwo ? +1 : -1;
+    };
+
+    const comparer = arguments.length === 0 ? defaultCompare : compareFunction;
```

Comment:
While using `arguments.length` is not incorrect, it is more common in modern JavaScript to define the parameter with a default value like `compareFunction = defaultCompare`. This makes the code cleaner and easier to understand.